### ISSUE TITLE
localhost can't currently handle the request

### ISSUE THREAD

**Issue by DataTraveler1** [ironmansoftware/powershell-universal#920]
### 1. Under Security -> Authentication. Click disable on authentication for the form type

![image](https://user-images.githubusercontent.com/537287/152669211-f92c9d18-9563-4ec9-b452-80cb26f517d7.png)

### 2. The page will change to unauthorized access

![image](https://user-images.githubusercontent.com/537287/152669242-acc95ea4-e22b-4180-b35c-be7fccdb6251.png)

### 3. Attempts to continue result with http error 500

> _This page isn't working right now - localhost can't currently handle the request._

![image](https://user-images.githubusercontent.com/537287/152669264-583c03b8-0114-4558-86de-74b553c19860.png)

### Expected behavior

```console
The application should not allow this kind of change or the mechanism needs to be modified to avoid breakage.
```

### 1. Use an **elevated** application (pictured below) to edit the file C:\ProgramData\UniversalAutomation\Repository\.universal\authentication.ps1

![image](https://user-images.githubusercontent.com/537287/152669409-d4102098-736d-42cd-b210-09965a545ae5.png)

### 2. Remove the " -Disabled" portion (10 bytes) at the bottom of the script (pictured below) and save it.

![image](https://user-images.githubusercontent.com/537287/152669446-9462e625-164b-4bba-8f00-2c6d53fdb15a.png)

:warning: If you receive access denied when trying to save the .ps1 file it's because you weren't elevated

### Environment data

PowerShell Universal 2.7.4 (fresh MSI install)
Windows 10 21H2

**Comment by DataTraveler1** [comment]
Note that this issue is not related to the similar one reported on the forums at https://forums.ironmansoftware.com/t/issue-with-authentication-ps1-file/6422 although this was a helpful post.

**Comment by adamdriscoll** [comment]
This has been resolved in PSU 2.8 We've improved the UI two-fold. First, I've added a confirmation when clicking the enable\disable switch. Second, I've added validation to prevent all authentication from being disabled.
